gateway-service/src/main/java/org/zowe/apiml/gateway/security/config/SecurityConfiguration.java

```diff
@@ -180,6 +181,13 @@ CorsConfigurationSource corsConfigurationSource() {
         return source;
     }
 
+    private void addCorsRelatedIgnoredHeaders() {
+        zuulProperties.setIgnoredHeaders(new HashSet<>(
+            Arrays.asList(("Access-Control-Request-Method,Access-Control-Request-Headers,Access-Control-Allow-Origin," +
+                "Access-Control-Allow-Methods,Access-Control-Allow-Headers,Access-Control-Allow-Credentials").split(","))
```

Comment:
isnt it better to split those strings manually?

Reply:
I am not sure, this way it is simpler for us to load them from configuration properties.

Reply:
Which I expect we will need to do at some point.

Reply:
ok then. is there reason to not put them in configuration now?

Reply:
Ultimately nope. I will add it later today.